client/src/_reducers/auth.js

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs'
+import jwt_decode from 'jwt-decode'
+import { ADD_AUTH, REMOVE_AUTH } from '_actions/auth'
+
+const initialAuthState = {
+  token: localStorage.getItem('pie-token'),
+  expiration: dayjs(localStorage.getItem('pie-expiration')) || dayjs()
```

Comment:
The`dayjs` function will always return an object — the second operand (`dayjs()`) will never get executed . Shouldn't it be the following? 🤔 

```
const token = localStorage.getItem('pie-token')
const expiration = localStorage.getItem('pie-expiration')
return {
  token,
  expiration: expiration ? dayjs(expiration) : dayjs()
}
```
```

Reply:
Ooo good point.  I'll fix it.